File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    const mySet = new Set(nums);
    return !(mySet.size == nums.length);
    // for (let i = 0; i < nums.length; i++ ) {
    //     if (mySet.has(nums[i])) {
    //         return true;
    //     }
    //     mySet.add(nums[i]);
    // }
    // return false;
};

File: 217-contains-duplicate/contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # mySet = set()

        # for num in nums:
        #     if num in mySet:
        #         return True
        #     mySet.add(num)
        # return False

        return not len(nums) == len(set(nums))